### ISSUE TITLE
[iOS] openPicker() fail with E_NO_IMAGE_DATA_FOUND after select an image

### ISSUE THREAD

**Issue by howg0924** [ivpusic/react-native-image-crop-picker#1489]
### Version

- react-native-image-crop-picker v0.35.1
- react-native v0.62.2

### Platform

iOS

### Expected behaviour

ImagePicker.openPicker() should return the selected image information.

### Actual behaviour

ImagePicker.openPicker() failed with error E_NO_IMAGE_DATA_FOUND after selecting an image

This only happened to a small part of iOS devices of our customers. We still don’t know what caused this.

### Steps to reproduce

It’s hard to reproduce. We cannot reproduce it in-house currently. All we got is customers’ complains and error code collected from their devices.

1.	call ImagePicker.openPicker() and select an image

2.	On some devices, it will failed with E_NO_IMAGE_DATA_FOUND

**Comment by lunajing** [comment]
@howg0924  Did you solved it？

**Comment by howg0924** [comment]
@lunajing No, I don't know how to solve it yet. Do you encounter the same problem? Have any idea what caused this?

**Comment by HugoGresse** [comment]
Duplicate of https://github.com/ivpusic/react-native-image-crop-picker/issues/1538

**Comment by howg0924** [comment]
@HugoGresse This is not an duplicate of #1538. #1538 is on Android. And I encountered the problem on iOS. I guess these two issues have different causes on native side.

**Comment by HugoGresse** [comment]
hmm, you're right, I believe the issue is with .heic, at this on the Camera side :/ 
Using Compatibility mode solved this but is not a logn term solution.

**Comment by TayyabaAslam3317** [comment]
@HugoGresse how di you resolve the issue?

**Comment by HugoGresse** [comment]
I believe the latest release fix this issue. I've not yet confirmed with my users though. 
https://github.com/ivpusic/react-native-image-crop-picker/pull/792

**Comment by HugoGresse** [comment]
it seems some users still have the issue, though I cannot find why it is happening.

**Comment by HugoGresse** [comment]
This should probably need more attention. Is does impact many users on our side.

**Comment by anhduy5689h** [comment]
This issue also occurred on v0.40.0, react-native 0.72.4. It occurred on iPad with software version 15.7.

**Comment by sregg** [comment]
We're having similar `E_NO_IMAGE_DATA_FOUND` error on both iOS and Android.
It'd be great to add more info to the error message like the filename to debug more.

**Comment by pckz** [comment]
any news with this issue? we believe is for images which are in icloud, but makes sense too for .heic photos

    "react-native-image-crop-picker": "^0.41.2",
    "react-native": "0.73.3",
